index.html

```diff
@@ -183,35 +183,31 @@ <h3 id="unacceptablebehaviour">
 		
-  <li>Never <a>harass</a> or <a>bully</a> anyone verbally, physically or 
-   <a data-lt="sexual harassment">sexually</a>.</li>
-  <li>Never <a>discriminate</a> on the basis of personal characteristics or 
-    group membership.</li>
-  <li>Communicate constructively and avoid <a>demeaning</a> or <a>insulting</a>
-    behavior or language.</li>
-  <li>Seek, accept, and offer objective work criticism, and <a>acknowledge</a> 
-    properly the contributions of others. </li>
-  <li>Be honest about your own qualifications, and about any circumstances that
-    might lead to <a href="/Consortium/Process/policies.html#coi">conflicts of
-    interest</a>.</li>
-	
-  <li>Promote the rules of this Code and take action (especially if you are in
-    a <a>leadership position</a>) to bring the discussion back to a more civil
-    level whenever inappropriate behaviors are observed.</li>
-</ol>
+		<li><strong>Be inclusive and Promote Diversity.</strong> Seek diverse perspectives. Diversity of views and of people powers innovation, even if it is not always comfortable. Encourage all voices. Help new perspectives be heard and listen actively. If you find yourself dominating a discussion, it is especially important to step back and encourage other voices to join in. Provide alternative ways to contribute.</li>
+		
+		<li>Be aware of how much time is taken up by dominant members of the group.  </li>
+		
+  <li>Be aware that displays of affection may complicate professional relationships. For some cultures, overtly friendly disposition towards another participant involving body contact (e.g.: hugging, touching on the arm or shoulder, or kissing) is uncommon and may be perceived as an invasion of personal space, or as unwelcome advances. Work to eliminate your own biases, prejudices and discriminatory practices. </li>
+
+<li>Think of others’ needs from their point of view. Use preferred names, titles (including pronouns) and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others, at least until a truly friendly atmosphere and relationships are established.</li>
+
+<li>Be sensitive to language differences. English is the default language of the W3C. However, only some of us are native English speakers. Many participants speak English as a second (third, …). People who communicate in non-native language often struggle to understand fast and/or quiet speech, and tend to speak louder than they usually would when communicating in their native tongue. If someone struggles to express his thoughts, help ensure their ideas are adequately expressed, heard, and granted thorough consideration.</li>
```

Comment:
typo: missing word after `(third...)`? Possibly the word `language`.

Reply:
The original text of the PWETF draft (from which this particular part was borrowed) does have the word "language" in that sentence: "Many participants speak English as a second (third, …) language, and may not possess the same level of fluency or eloquence required to express complex concepts and ideas."

Reply:
The sentence of the PWETF draft I just quoted is immediately followed by "People who communicate ..." so I am now wondering if the whole part of that sentence has been accidentally dropped as a result of botched copy/paste operation.

Reply:
wording corrected

Reply:
Sorry @TzviyaSiegman I can't see any commit that fixes the wording for example by adding the word "language". Is the fix pushed?

Reply:
This was fixed in a later commit by @AdaRoseCannon .